config/rule.go

```diff
@@ -82,6 +85,48 @@ func (r *Rule) Inspect(line string) *Offender {
 	}
 }
 
+// InspectFile checks the content of the file for  multilines leaks
+func (r *Rule) InspectFile(fileLines string) []Offender {
+	fileLines = regexp.MustCompile(`[\t\r\n]+`).ReplaceAllString(strings.TrimSpace(fileLines), "\n")
```

Comment:
A recent multi-line regex should handle `\t\r\n` fine. This manipulates every line of prior to checking the regex which makes it very expensive on larger files.

Reply:
Not sure what you mean by 👎 , but this is unnecessary, inefficient, and leads to errors and false negatives/positives. Automatically replacing all tabs with newlines will cause absolute chaos in tab-delimited files. We should not manipulate the content and instead recommend building good regex. Instead of filtering/manipulating `\t\r\n`, you can recommend people use `\s` (≡ `[\t\n\f\r]`) per [re2 docs](https://github.com/google/re2/wiki/Syntax). 

Example:
```
func something
\t\t\telse
```
If someone wants to write a regex:
`something\n\t\t\telse` (does not match, false negative)
`something\n\t+else` (does not match, false negative)
`something\n\s+else` (does match, true positive)
`something\n\n\n\nelse` (does match, false positive)

However, this would also match:
```
something



else
```
Which is a false positive.